src/nnet3/nnet-test-utils.cc

```diff
@@ -793,7 +793,7 @@ void ComputeExampleComputationRequestSimple(
 
 static void GenerateRandomComponentConfig(std::string *component_type,
                                           std::string *config) {
-  int32 n = RandInt(0, 20);
+  int32 n = RandInt(0, 21);
```

Comment:
OK thanks.
So you've addressed all the issues we raised?


On Wed, Nov 4, 2015 at 1:25 PM, hhadian <notifications@github.com> wrote:

> In src/nnet3/nnet-test-utils.cc
> <https://github.com/kaldi-asr/kaldi/pull/307#discussion_r43917941>:
>
> > @@ -793,7 +793,7 @@ void ComputeExampleComputationRequestSimple(
> >
> >  static void GenerateRandomComponentConfig(std::string *component_type,
> >                                            std::string *config) {
> > -  int32 n = RandInt(0, 20);
> > +  int32 n = RandInt(0, 21);
>
> I scaled the output and the derivative test failed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/307/files#r43917941>.
>